R/labs_e61.R

```diff
@@ -49,54 +49,157 @@ labs_e61 <- function(title = NULL,
                      subtitle = NULL,
                      footnotes = NULL,
                      sources = NULL,
-                     title_max_char = 100,
-                     subtitle_max_char = 120,
-                     footnote_max_char = 140,
-                     title_wrap = TRUE,
-                     subtitle_wrap = TRUE,
-                     footnote_wrap = TRUE,
+                     title_max_char = NULL,
```

Comment:
Checking my understanding. This restores the ability to set the max characters, but not the ability to turn off wrapping entirely (that was present before?)

Reply:
Well you can just set max char to 1e10 or something to effectively turn it off.